packages/account/src/Components/currency-selector/radio-button-group.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Localize } from '@deriv/translations';
+
+// Radio group
```

Comment:
Component name sounds self explanatory, is there a reason this is here?

Reply:
Removed it 👍